src/pages/AccountSettings/tabs/Admin/DeletionCard/DeletionCard.tsx

```diff
@@ -0,0 +1,45 @@
+import Card from 'old_ui/Card'
+import A from 'ui/A'
+
+interface EraseSectionProps {
+  isPersonalSettings: boolean
+}
+
+function EraseSection({ isPersonalSettings }: EraseSectionProps) {
```

Comment:
What is the difference between this and the other `DeletionCard` component in the profile tab? Do we want to update that too?

Reply:
Hrmrmrm profile tab is self hosted only, let me go ask in the ticket.

Reply:
They wanted it on both